integrations/virustotal.py

```diff
@@ -190,109 +174,153 @@ def request_virustotal_info(alert: any, apikey: str):
         string      = '1:{0}->virustotal:{1}'.format(location, json.dumps(msg))
 
-    debug("# Request result from VT server: %s" % string)
+    logger.debug("Event: %s", string)
+
     try:
         sock = socket(AF_UNIX, SOCK_DGRAM)
         sock.connect(SOCKET_ADDR)
         sock.send(string.encode())
         sock.close()
     except FileNotFoundError:
-        debug("# Error: Unable to open socket connection at %s" % SOCKET_ADDR)
+        logger.error("Error: Unable to open socket connection at %s", SOCKET_ADDR)
         sys.exit(ERR_SOCKET_OPERATION)
 
 def get_json_alert(file_location: str) -> any:
-    """
-        Read JSON alert object from file
-
-        Parameters
-        ----------
-        file_location : str
-            Path to the JSON file location.
-
-        Returns
-        -------
-        {}: any
-            The JSON object read it.
-
-        Raises
-        ------
-        FileNotFoundError
-            If no JSON file is found.
-        JSONDecodeError
-            If no valid JSON file are used
+    """Read JSON alert object from file
+
+    Parameters
+    ----------
+    file_location : str
+        Path to the JSON file location.
+
+    Returns
+    -------
+    any
+        JSON encoded alert.
+
+    Raises
+    ------
+    FileNotFoundError
+        If no JSON file is found.
+    JSONDecodeError
+        If no valid JSON file is used.
     """
     try:
-        with open(file_location) as alert_file:
+        with open(file_location, encoding='utf-8') as alert_file:
             return json.load(alert_file)
     except FileNotFoundError:
-        debug("# JSON file for alert %s doesn't exist" % file_location)
+        logger.error("JSON file for alert %s doesn't exist", file_location)
         sys.exit(ERR_FILE_NOT_FOUND)
     except json.decoder.JSONDecodeError as e:
-        debug("Failed getting JSON alert. Error: %s" % e)
+        logger.error("Failed getting JSON alert. Error: %s", e)
         sys.exit(ERR_INVALID_JSON)
 
+def print_help_msg():
+    help_msg = '''Exiting: Invalid arguments.
+    
+Usage:
+    virustotal <alerts_file> [api_key] <webhook_url> [logging_level] [options_file]
+Arguments:
+    alerts_file (required)
+        Path to the JSON file containing the alerts.
+    api_key (required)
+        Virus total API key.
+    webhook_url (not required)
+        The webhook URL argument is not needed for the Virus total integration. However, it's still considered because the 
+        integrator executes all scripts with the same arguments.
+        If you are executing the script manually, please put anything in that argument.
+    logging_level (optional)
+        Used to define how much information should be logged. Default is INFO.
+        Levels: NOTSET, DEBUG, INFO, WARNING, ERROR, CRITICAL.
+    options_file (optional)
+        Path to a file containing custom variables to be used in the integration. It must be JSON-encoded.
+    '''
```

Comment:
```suggestion
    Exiting: Invalid arguments.
    
    Usage:
        virustotal <alerts_file> [api_key] <webhook_url> [logging_level] [options_file]
    Arguments:
        alerts_file (required)
            Path to the JSON file containing the alerts.
        api_key (required)
            Virus total API key.
        webhook_url (not required)
            The webhook URL argument is not needed for the Virus total integration. However, it's still considered because the 
            integrator executes all scripts with the same arguments.
            If you are executing the script manually, please put anything in that argument.
        logging_level (optional)
            Used to define how much information should be logged. Default is INFO.
            Levels: NOTSET, DEBUG, INFO, WARNING, ERROR, CRITICAL.
        options_file (optional)
            Path to a file containing custom variables to be used in the integration. It must be JSON-encoded.
    '''
```